src/librustdoc/test.rs

```diff
@@ -64,6 +64,9 @@ pub fn run(options: Options) -> i32 {
 
     let mut cfgs = options.cfgs.clone();
     cfgs.push("rustdoc".to_owned());
+    if options.should_test {
```

Comment:
You can remove the `options.should_test` check here because it will always be `true` in this function.

Reply:
Ah indeed, good catch!